containers/ddev-xhgui/Dockerfile

```
@@ -1,20 +1,4 @@
-FROM xhgui/xhgui:0.21 AS xhgui-docker
-
-FROM scratch AS ddev-xhgui
-
-### --------------------------------xhgui-docker----------------------------------
-### XHGui Docker image creates a volume for /run/nginx, which we want to remove.
-### We copy all files from xhgui/xhgui, and set ENV, WORKDIR, CMD, EXPOSE.
-### Source https://github.com/perftools/xhgui/blob/0.21.x/Dockerfile
-COPY --from=xhgui-docker / /
-ENV PHP_INI_DIR=/etc/php7
-ARG APPDIR=/var/www/xhgui
-ARG WEBROOT=$APPDIR/webroot
-WORKDIR $APPDIR
-RUN mkdir -p /run/nginx
-CMD ["sh", "-c", "nginx && exec php-fpm"]
-EXPOSE 80
-### ------------------------------END xhgui-docker--------------------------------
+FROM xhgui/xhgui:0.23 AS ddev-xhgui
```

```suggestion
FROM xhgui/xhgui:0.23.0 AS ddev-xhgui
```

shouldn't you use actual tag? 0.23 is latest 0.23.x release, i.e it's ovewritten with next release

---

I think it's fine to use 0.23 since 0.23.x will be minor releases that shouldn't break anything (plus, we have tests).

This way, we don't need to constantly remember to update it here. (And we handle Traefik in the same way.)

---

We are usually OK with following a branch (movable tag, 0.23), especially since we build and maintain a derivative image which is only updated at our release time.  And of course... when Stas did this there wasn't an 0.23.0 :)

---

xhgui doesn't guarantee any semver releases (no manpower). besides by semver, in 0.x release series any release is allowed to add a breaking change. 

and of course any change can break someone's workflow: 
- https://xkcd.com/1172/

---

We're okay with this, and it could actually be a win-win for both of us to catch any bugs as early as possible.